.gitignore

```diff
@@ -2,4 +2,7 @@ node_modules/
 .DS_Store
 data/
 server/config.js
-coverage/
\ No newline at end of file
+coverage/
+server/db/query.sql
+newrelic_agent.log
+newrelic.js
```

Comment:
I think you still need this `newrelic.js` file in your deployed instance. If you don't want to manually add this file again to the deployed version, you can include it in your git repo and use a environment variable to hide your license key.